src/main/scala/mesosphere/marathon/BuildInfo.scala

```diff
@@ -3,20 +3,20 @@ package mesosphere.marathon
 import java.util.jar.{Attributes, Manifest}
 import scala.Predef._
 import scala.util.control.NonFatal
-import mesosphere.marathon.stream.Implicits._
+import scala.jdk.CollectionConverters._
 import mesosphere.marathon.io.IO
 
 case object BuildInfo {
   private val marathonJar = "\\bmesosphere\\.marathon\\.marathon-[0-9.]+".r
-  val DefaultBuildVersion = SemVer(1, 9, 0, Some("SNAPSHOT"))
+  val DefaultBuildVersion = SemVer(1, 10, 0, Some("SNAPSHOT"))
```

Comment:
We need to cut a branch before this lands.

Reply:
Agreed, I'll do it right before merging